packages/web-lib/index.ts

```diff
@@ -1,3 +1,3 @@
 export { Logger } from './src/logger/logger';
-export { WebStore } from './src/persistence/web-store';
 export * from './src/network';
+export { StorageContext } from './src/persistence/web-store-manager';
```

Comment:
it would be great to rename StorageContext to WebStoreManager. StorageContext is correct name for underlying design pattern, but not really descriptive for consumers.